features/keychain/module/crypto/ed25519.js

```diff
@@ -12,6 +13,10 @@ export const create = ({ getPrivateHDKey }) => {
       const { sign } = await getSodiumKeysFromIdentifier({ seedId, keyId })
       return sodium.signDetached({ message: data, privateKey: sign.privateKey })
     },
+    getPublicKey: async ({ seedId, keyId }) => {
+      const { sign } = await getSodiumKeysFromIdentifier({ seedId, keyId })
```

Comment:
Doesn't `exportKey` return the same public key for ed25519?

Reply:
Not sure? Doesn;t `sodium.getSodiumKeysFromSeed(sodiumSeed)`  create a new keypair? If not, you may be right.

Reply:
looks the same

https://github.com/ExodusMovement/exodus-oss/blob/97beaf359a9a0bdabff4526bee3110781baf5c69/features/keychain/module/keychain.js#L77-L78
https://github.com/ExodusMovement/exodus-oss/blob/97beaf359a9a0bdabff4526bee3110781baf5c69/features/keychain/module/keychain.js#L91
https://github.com/ExodusMovement/exodus-oss/blob/82e5cf2e1b9334af150bf2a8bba4abbc0f74c764/features/keychain/module/crypto/sodium.js#L81-L86

Reply:
We do not want `privToPub` involved in getting the public key. It would defeat the purpose of having an asset-agnostic signer concept.